<launch>
    <node name="capo2_camera_publisher"
          pkg="topnav_capo2"
          type="capo2_camera_publisher.py"
          respawn="false" output="screen"/>
</launch>
